app/models/event.rb

```diff
@@ -35,8 +35,6 @@ class Event < ActiveRecord::Base
   validates :program, presence: true
   validates :max_attendees, numericality: { only_integer: true, greater_than_or_equal_to: 1, allow_nil: true }
 
-  validate :max_attendees_no_more_than_room_size
```

Comment:
> The problem with removing this from here is that when a proposal is scheduled in a room that can have up to 10 people, when the submitter edits the proposal, can set max_attendees to 15, and enable require_registration. Which will allow 15 people to register to a session that takes place in a room that can only hold only 10 people.

Maybe I should keep the validation in both event and event schedule.

> we should not break scheduling because of this. An organizer wants to schedule a specific session into a specific room at a specific time, a warning would be excellent, but disallowing it all together, I don't know. Mainly because the organizer cannot properly remedy this. Once 15 people have registered to attend, the organizer cannot delete the registrations.

For me allowing the organizer to schedule a talk with can have 15 attendees if there is only place for 10 has no sense. If it is possible that this talk take place in that room then the size of the room should be increased, and the organizer can do that before scheduling the event.

> the workflow of max_attendees is a subject for the proposal registration process to solve, not this PR. Unless there is something really failing with your changes. Is there?

No, nothing is failing. I moved it because I thought that we need to validate all event schedules, I didn't take into account that the max attendees can be change after scheduling. I can keep validations in both models and we can discuss afterward (outside this PR) if we want to change the way it works (validate less things or in a different way).